docs/contributing/making-changes-to-a-pull-request.md

```diff
@@ -0,0 +1,17 @@
+# Making changes to a Pull Request
+
+When a Pull Request branch is [checked out](review-a-pull-request-in-visual-studio.md) you can commit changes to it and push and pull like any other branch. If the Pull Request branch is located in a fork and was checked out from the GitHub Pull Request details pane, then a remote to that fork will be created automatically and the branch set to track the fork branch.
```

Comment:
What I was trying to refer from was the branch that the Pull Request was created from. Is that what "topic branch" means? I've never heard this terminology, but we can switch to that if it'd be better.